app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -219,9 +244,49 @@
     },
     attributes = Map("rows" -> "5", "autofocus" -> "true")))
 
+
+  <label class="govuk-label">@messages("edit-set.record.related-material")</h3>
+  <ul class="govuk-list govuk-list--bullet">
+    @{
+      staticRecord.relatedMaterial.map { entry =>
+        entry.description match {
+
+          case Some(description) => {
+            relatedMaterialItemWithDescription(entry.linkHref, entry.linkText, description)
+          }
+
+          case None => {
+            relatedMaterialItem(entry.linkHref, entry.linkText)
+          }
+
+        }
+      }
+    }
+  </ul>
+
+  <label class="govuk-label">@messages("edit-set.record.edit.separated-material")</h3>
```

Comment:
Rather than a label this should be a h3 as per the prototype, e.g. `<h3 class="govuk-heading-s">Related material</h3>`